lib/carrierwave/orm/activerecord.rb

```diff
@@ -43,6 +43,7 @@ def reload(*)
         # Reset cached mounter on record dup
         def initialize_dup(other)
           old_uploaders = _mounter(:"#{column}").uploaders
+          @_mounters =  @_mounters.dup
```

Comment:
Thank you for the PR! Your fix works, but let me propose a slight modification...

As was discussed in https://github.com/carrierwaveuploader/carrierwave/issues/1962#issuecomment-1370308875, this will be executed for every `mount_uploader` / `mount_uploaders`, not just once for a ActiveRecord instance. If we move this to `CarrierWave::Mount::Extension` like 

```diff
--- a/lib/carrierwave/mount.rb
+++ b/lib/carrierwave/mount.rb
@@ -428,6 +428,11 @@ module CarrierWave

     private

+      def initialize_dup(other)
+        @_mounters =  @_mounters.dup
+        super
+      end
+
       def _mounter(column)
         # We cannot memoize in frozen objects :(
         return Mounter.build(self, column) if frozen?
```

I think we can resolve that mismatch. What do you think?

Reply:
Thanks for your suggestion! This looks like a better solution.